package.json

```diff
@@ -107,5 +107,6 @@
         "vite-tsconfig-paths": "^5.1.4",
         "vitest": "*",
         "vitest-monocart-coverage": "^3.0.0"
-    }
+    },
+    "overrides": {}
```

Comment:
do we need the `overrides`?

Reply:
lol, nope.  I thought I might for the qrcode lib but then the warnings went away